nats/js/manager.py

```diff
@@ -68,12 +68,13 @@ async def find_stream_name_by_subject(self, subject: str) -> str:
             raise NotFoundError
         return info['streams'][0]
 
-    async def stream_info(self, name: str) -> api.StreamInfo:
+    async def stream_info(self, name: str, subjects_filter: str = '>') -> api.StreamInfo:
```

Comment:
I think we want `subjects_filter` to be an optional parameter by default so something like:
```suggestion
    async def stream_info(self, name: str, subjects_filter: str = None) -> api.StreamInfo:
```